openassessment/xblock/data_layer/serializers.py

```diff
@@ -0,0 +1,201 @@
+from rest_framework.serializers import (
+    BooleanField,
+    DateTimeField,
+    IntegerField,
+    Serializer,
+    CharField,
+    ListField,
+    SerializerMethodField,
+)
+
+
+class CharListField(ListField):
+    child = CharField()
+
+
+class TextResponseConfigSerializer(Serializer):
+    enabled = SerializerMethodField()
+    required = SerializerMethodField()
+    editor_type = CharField(source="text_response_editor")
+    allow_latex_preview = BooleanField(source="allow_latex")
+
+    def get_enabled(self, block):
```

Comment:
I wish there was a simple way to remove this semi-duplicated code across the few serializers but i think doing that would complicate things more than they need to be